hypothesis-python/tests/cover/test_float_nastiness.py

```diff
@@ -251,3 +251,13 @@ def test_no_single_floats_in_range():
         with warnings.catch_warnings():
             warnings.simplefilter("ignore")
             st.floats(low, high, width=32).validate()
+
+
+@given(st.floats(min_value=1e304, allow_infinity=False))
+def test_finite_min_bound_does_not_overflow(x):
+    assert not math.isinf(x)
+
+
+@given(st.floats(max_value=-1e304, allow_infinity=False))
```

Comment:
Why choose `1e304` for these tests? A comment behind the choice would be helpful.

Reply:
Done - it's a large enough value that we reliable overflow if that's not explicitly handled.